src/router/handler.rs

```diff
@@ -26,7 +26,26 @@ where
         if let Some(headers) = response.headers() {
             if let Some(response_headers) = res.headers_mut() {
                 headers.iter().for_each(move |(key, value)| {
-                    response_headers.insert(key, header::HeaderValue::from_static(value));
+                    if let Ok(val_bytes) = header::HeaderValue::from_bytes(value.as_bytes()) {
+                        response_headers.insert(key, val_bytes);
+                    }
+                });
+            }
+        }
+
+        if let Some(cookies) = response.cookies() {
+            if let Some(response_headers) = res.headers_mut() {
+                cookies.iter().for_each(move |cookie| {
```

Comment:
How many cookies are there to iterate? I thought there should be only one cookie?

Reply:
`set-cookie` header can be multiple.